libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.ts

```diff
@@ -86,9 +91,10 @@ export class FormFieldComponent {
   @ViewChild('titleInput') titleInput: ElementRef
 
   isHidden = false
+  isEditing = false
 
   focusTitleInput() {
```

Comment:
I guess the name of the function is now misleading. Maybe something like `toggleEditing` would be more fitting?
But I am wondering if this has to be changed at all.
When testing the feature I can no longer click the "pen"-icon to edit the title.

Reply:
Good point about the name. The pen icon should be working, so it's not normal, I'll have a look at it. I think that by trying to simplify the feature, I lost sight of it

Reply:
So I ended up keeping the name `focusTitleInput` because I reverted my changes on that function. Now it should work with the pen icon like it did before :)